src/views/v2/rsvp-kitchen-sink/form-going.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Block: RSVP form going.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp-kitchen-sink/form-going.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: f66c0e43bda1f2ff36a6eff6c3535dfb</sub>